presto-hive/src/main/java/com/facebook/presto/hive/SortingFileWriter.java

```diff
@@ -146,8 +149,10 @@ public void commit()
     @Override
     public void rollback()
     {
-        for (TempFile file : tempFiles) {
-            cleanupFile(file.getPath());
+        if (!sortedWriteToTmpPathEnabled) {
```

Comment:
note: This can be very confusing. We should not forget to clean it up once the feature is rolled out.

Reply:
Yes, that was my plan as well. Let me add TODO around in the config.